public class Exception9 {
    public static void main(String[] args) {
        try {
            try {
                String[] array = {"A", "B", "C"};
                System.out.println(array[3]);
            } catch 
        }
    }
}
